app/handlers.mjs

```diff
@@ -19,7 +19,7 @@ export const summary = async (modbusClient, req, res) => {
         'readings': await getReadings(modbusClient),
         'settings': await getSettings(modbusClient),
         'deviceInformation': await getDeviceInformation(modbusClient),
-        'alarms': await getAlarmStatuses(modbusClient, true, false),
+        'alarmHistory': await getAlarmHistory(modbusClient, true, false),
```

Comment:
As you removed the need for onlyActive and distinct, the true, false should be removed from here.

Reply:
Good catch!

Reply:
Fixed